io.github.harmonoid.yml

```diff
@@ -0,0 +1,88 @@
+app-id: io.github.harmonoid
```

Comment:
```suggestion
app-id: io.github.prateekmedia.harmonoid
```

And files need to be renamed accordingly. (you don't have github.com/harmonoid )

Reply:
oops yes

Reply:
actually no, I will give it to upstream and it's not my project.

Reply:
I have contacted upstream regarding this

Reply:
io.github.harmonoid.harmonoid then.

But why do you download from your fork?

Reply:
I downloaded from fork temporarily until this is a PR is accepted upstream.

Reply:
@hfiguiere can you figure out why libmediainfo couldn't build?

Reply:
Also I think io.github.harmonoid.harmonoid would be a repetition which may not be good id, also if someone like xyz wants to publish his version then it can keep id as io.github.xyz.harmonoid, so I would keep it as is.

Reply:
it is not free form. If upstream is in github.com/harmonoid and the app is harmonoid then `io.github.harmonoid.harmonoid` is right.